src/app/legacy/containers/ServiceWorker/index.jsx

```diff
@@ -15,7 +38,22 @@ const ServiceWorkerContainer = () => {
     }
   }, [envIsProduction, swPath, service]);
 
-  return null;
+  return isAmp && process.env.SIMORGH_BASE_URL.indexOf('http:') === -1 ? (
```

Comment:
both. Serviceworkers can only run on https. But I suspect local is only way this would happen in reality. You think we should just check for running locally?

Reply:
Yea just wasn't sure if it was possible to get to `http` other than through `localhost`, thought Fastly/GTM/Mozart/Belfrage would prevent that. The check `process.env.SIMORGH_APP_ENV !== 'local'` may be better if that is the case.

Reply:
changed